glyph/src/ttfp.rs

```diff
@@ -104,6 +104,12 @@ impl fmt::Debug for FontVec {
     }
 }
 
+impl Clone for FontVec {
+    fn clone(&self) -> Self {
+        Self::try_from_vec(self.as_slice().to_vec()).unwrap()
```

Comment:
I think this impl means that mutations, e.g. setting variations, will not be present in the "cloned" output which would be incorrect.

Reply:
Well, I have no better ideas. Maybe byte-to-byte clone?